.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+name: Release
+on:
+  push:
+    tags: '*'
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  release:
+    runs-on: windows-2019 # Code signing requirement https://github.com/NuGet/Home/issues/7939
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - name: Parse repo name
+        run: |
+          $FullName = "$env:GITHUB_REPOSITORY"
+          $Org,$RepoName = $FullName.Split('/')
+          echo "PARTICULAR_REPO_NAME=$RepoName" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        shell: pwsh
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: 5.0.x
+      - name: Setup NuGet for signing
+        uses: nuget/setup-nuget@v1.0.5
+        with:
+          nuget-api-key: ${{ secrets.NuGetAPIKey }}
+      - name: Build
+        run: |
+          cd src
+          dotnet build --configuration Release
+        shell: pwsh
+      - name: Get signing cert
+        run: |
+          [IO.File]::WriteAllBytes("signing-cert.pfx", [Convert]::FromBase64String("${{ secrets.NUGET_SIGNING_CERT_BASE64 }}"))
```

Comment:
Are the `|` and line break required?

Reply:
If you don't, the fact that it requires special characters like [ and ] and : becomes a problem, especially since it also contains double quotes which makes quoting it hard. YAML and PowerShell don't exactly like each other.